### DISCUSSION TITLE
Sponsor

### DISCUSSION THREAD

**Discussion by TesterAchievement** [community/community#144715] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

how can i get the badge

**Comment by Enrico-github** [comment, marked as answer]
Set Up a GitHub Sponsors Profile:

    Navigate to [GitHub Sponsors](https://github.com/sponsors).
    Click on "Get started with GitHub Sponsors".
    Fill out the application form with details about your open-source contributions and projects.
    You may need to provide personal or organization details and agree to the program's terms and conditions.

Wait for Approval:

    After submitting your application, GitHub will review your request to ensure you meet the program's eligibility criteria.
    Approval times vary, but you’ll receive a notification once approved.

Set Up Your Sponsorship Tiers:

    After approval, configure your sponsorship tiers with descriptions and suggested amounts (e.g., $5/month for shout-outs, $50/month for custom support).
    Make your profile compelling by describing why others should sponsor your work.

Enable Your Sponsor Button:

    Once you’re approved, a "Sponsor" button will appear on your GitHub profile and repositories.
    You can also add the sponsor badge to your README files or other locations.

Promote Your Sponsorship:

    Share your GitHub Sponsors link on social media, blogs, or within the open-source community to attract sponsors.
